### ISSUE TITLE
Icon dissapeared

### ISSUE THREAD

**Issue by NoNamePaul** [brittyazel/Neuron#103]
I have two almost identical buttons, but the icon is only showing for one of them:
1.  `/cast [@focus] Moonfire`
2.  `/cast [@focus] Sunfire`
The icon only shows for moonfire (I tried swapping buttons, same result)
The buttons works though and the icon was showing yesterday (before the recent update)

**Comment by brittyazel** [comment]
use these instead:
/cast [@focus] Sunfire(solar)
/cast [@focus] Moonfire(lunar)

**Comment by NoNamePaul** [comment]
Your solution works perfectly fine for all circumstances i tested it.
One thing though I just noticed during testing of issue #102 that might help understanding where the "error" comes from and why I didn't notice it earlier:
The macros I used above in the original post work fine (initially and after relogs) until i change speccs and come back into balance. Then the icon is gone, but yet again this is only the case for the sunfire.
Moonfire works perfectly fine even without the "(lunar)".
